### ISSUE TITLE
Sonata show tab : 'Variable "view_group" does not exist.'

### ISSUE THREAD

**Issue by Karimgasmi47** [sonata-project/SonataAdminBundle#4531]
Hi there,

Since the execution of the composer update this morning to get the sonata-project/admin-bundle (3.20.0) version, I got the following error when I try to access to 'show' views which contain tab (the others 'show' views works well :

```
Variable "view_group" does not exist.
500 Internal Server Error - Twig_Error_Runtime
```

![erreur_twig](https://user-images.githubusercontent.com/22818120/27484278-f7cdaa62-5828-11e7-80a6-286ceed0dcee.png)

I'm a very beginner of both Sonata and Symfony. I suppose that something changed on the views during last update but don't know how to clearly correct it.

The log error is : 
`CRITICAL - Uncaught PHP Exception Twig_Error_Runtime: "Variable "view_group" does not exist." at /home/project/sonataProject/vendor/sonata-project/admin-bundle/Resources/views/CRUD/base_show.html.twig line 47 `

#### PHP version

PHP 7.0.15

## Steps to reproduce
Create a show view (with configureShowFields) which contain tab like : 

```
$showMapper
                ->tab('Car')
                ->with('Car', array('class' => 'col-md-6'))
                    ->add('num', "text", array('label' => "Number"))
                ->end()
->end()
```

## Expected results
The views with tab. 

## Actual results
Variable "view_group" does not exist.

On the whole stack trace, there is the code of all twig templates so I can't past it here.

Thanks for reading and don't hesitate if something is missing on the request,

Sincerely,

Hidden.

**Comment by jlamur** [comment]
This has already been discussed and fixed here: #4525. Rollback to previous version or wait for the next release.

**Comment by Karimgasmi47** [comment]
Ok thx, didn't saw it on my research
